simplecpp.cpp

```diff
@@ -1737,16 +1737,10 @@ namespace {
         }
     }
 
+    const char * const altopData[] = {"and","or","bitand","bitor","not","not_eq","xor"};
+    const std::set<std::string> altop(&altopData[0], &altopData[7]);
```

Comment:
As I see, cppcheck requires c++11, so is it allowed to use c++11 features in simplecpp?
There are so many beautiful features in c++11.